client/extensions/woocommerce/app/settings/payments/payment-method-stripe.js

```diff
@@ -36,9 +37,19 @@ class PaymentMethodStripe extends Component {
 		onCancel: PropTypes.func.isRequired,
 		onEditField: PropTypes.func.isRequired,
 		onDone: PropTypes.func.isRequired,
+		site: PropTypes.shape( {
+			title: PropTypes.string,
+		} ),
 	};
 
 	onEditFieldHandler = ( e ) => {
+		// Limit the statement descriptor field to 22 characters
+		if ( e.target && e.target.name && 'statement_descriptor' === e.target.name ) {
```

Comment:
Fixed in https://github.com/Automattic/wp-calypso/pull/16774/commits/72de550e887a557ad2e4610c6e5ee1c6da3c9c3d